earthly/rust/tools/Earthfile

```diff
@@ -70,10 +70,9 @@ tool-wasm-tools:
 tool-cargo-expand:
     DO +CARGO_BINSTALL --package=cargo-expand --version=1.0.90
 
-# Should align with our wit-bindgen library, but doesn't currently or it fails to build.
-# DO NOT change the version without testing that hermes targets all build correctly with the updated tool.
+# Should align with our wit-bindgen library.
 tool-wit-bindgen:
-    DO +CARGO_BINSTALL --package=wit-bindgen-cli --version=0.29.0 --executable="wit-bindgen" --test_param="-V"
+    DO +CARGO_BINSTALL --package=wit-bindgen-cli --version=0.43.0 --executable="wit-bindgen" --test_param="-V"
```

Comment:
Can we pin to hash `ffce1f7069b76986b1539b8136e141ecd4ff3334` so it picks up the markdown docs fixes PR?
its still version `0.43.0`.

Reply:
I guess not without serious effort because cargo binstall can't do that.